### ISSUE TITLE
Error message despite successful connection of Umbrel node

### ISSUE THREAD

**Issue by MoritzKa** [getAlby/lightning-browser-extension#1642]
### Describe the bug

Alby can connect to an Umbrel node although there is a error message

### Screenshots of the changes [optional]
![slack_n4X4yAoERn](https://user-images.githubusercontent.com/13707997/196777403-0ce5b233-e6e0-45dc-b746-281b28dcd91e.png)
![slack_pIQalev69n](https://user-images.githubusercontent.com/13707997/196777333-74c93372-0a1b-4ab1-bdcd-fd0414831e41.png)
![slack_wNqowLi8GT](https://user-images.githubusercontent.com/13707997/196777303-0a9273a4-9bbb-4db4-8303-83bdf40a2e97.png)

### To Reproduce

Connect Umbrel node via lnd connect to Alby

### Expected behavior

no error message when the connection was successful

### Alby information

- Alby Version: n.a.
- Alby installed through n.a.
- Wallet connected with Alby: Umbrel

### Device information [optional]

- OS: Mac Ventura OS 13.0
- Chrome Version 106.0.5249.119.

### Are you working on this issue?

No

**Comment by bumi** [comment]
that does not look successful... does it show a balance?

**Comment by jgertler88** [comment]
Yes, it shows a balance after about a minute or so.

**Comment by jgertler88** [comment]
So, I just removed my Umbrel and redid the set up process again and I received a successful process.  It had given me connection errors twice before when I tried adding.

**Comment by jgertler88** [comment]
Just redid it again and got a connection error.  Will send screenshot of error and Alby with balance showing.

**Comment by jgertler88** [comment]
<img width="789" alt="CleanShot 2022-10-19 at 14 00 35@2x" src="https://user-images.githubusercontent.com/71850419/196780655-abe53316-db90-4633-8fd0-d01d1cd5fcf0.png">

**Comment by jgertler88** [comment]
Did it one more time and got a fully successful.  So it just seems to be throwing off the connection errors, both alone and with a successful, even though the Umbrel node does successfully get reached.

<img width="676" alt="CleanShot 2022-10-19 at 14 02 55@2x" src="https://user-images.githubusercontent.com/71850419/196781196-1ff65206-5605-4985-bba9-87360693c487.png">

**Comment by MoritzKa** [comment]
let´s observe if it happens more often. 
I had a similar experience with my RaspiBlitz. Sometimes the connection worked and sometimes it did not despite the credentials have been the same.

**Comment by bumi** [comment]
do you connect through Tor? it is likely an issue with Tor being slow. After a while Alby shows that error assuming the connection fails. 

Sadly the Tor network is currently still under attack and sometimes unreliable.

**Comment by jgertler88** [comment]
Yes, I do connect through Tor so that is likely the issue.  If Alby knows it's trying to connect through Tor, should a dialog box pop up that says something about the issue with Tor, or should it allow a longer time-out period before throwing up an error?  It seems that given that I did receive a fully successful connection once that it only needs to extend the time-out for a little bit longer.  If anything, even just throwing up a quick error explanation about issues with Tor and to try again in a little bit would be helpful just to know that it's probably nothing wrong with my set-up or Umbrel node.

**Comment by bumi** [comment]
true, that for sure would help and the error should be removed again when it eventually works. 

thanks for raising this. let's take this issue to fix this.
